dev/ant_build.js/public_imports/internal_imports/jshint.xml

```diff
@@ -24,8 +24,8 @@
              classpath="${basedir}/../ant_build.js/lib/jshint/ant-jshint-0.3.6-deps.jar" />
 
     <!-- Lint the code -->
-    <!-- Default to running jshint unless the build definition overrides this property with false. -->
-    <property name="run.jshint" value="true" />
+    <!-- Default to skipping jshint unless the build definition overrides this property with true. -->
+    <property name="run.jshint" value="false" />
```

Comment:
Do build definitions override this today?  It would be good to have someone from the UI team review these changes as well.

Reply:
I've reverted these UI changes in hopes of finding a fix that allows them to still be run in parallel. I also don't have the issue trace, so I am hoping to recreate the problem.